gravity.sh

```diff
@@ -359,8 +359,18 @@ gravity_DownloadBlocklistFromUrl() {
     echo -ne "  ${INFO} ${str} Pending..."
     cmd_ext="--resolve $domain:$port:$ip $cmd_ext"
   fi
+
+  # Use compression to reduce the amount of data that is transfered
+  # between the Pi-hole and the ad list provider. Use this feature
+  # only if it is supported by the locally available version of curl
+  if curl -V | grep -q "Features:.* libz"; then
```

Comment:
Since this is adding a conditional feature that is dependent upon compile time options can we have a test added that makes sure this works when the feature is compiled in and when it is not compiled in?